vizro-ai/docs/pages/tutorials/explore-vizro-ai.md

```diff
@@ -1,29 +1,23 @@
 # Explore Vizro-AI
-This tutorial serves as an exploration of the extended applications offered by Vizro-AI beyond the initial [quickstart guide](../tutorials/quickstart.md).
-
-By the end of this tutorial, you will have gained an understanding of different language options and leveraging Vizro-AI to enhance the formatting your visualizations.
-
-## Let's get started!
-### 1. Install Vizro-AI and get ready to run your code
-Before proceeding, ensure the installation of the `vizro_ai` package by following the steps outlined in the [installation guide](../user-guides/install.md). Once installed, you can execute your code either by pasting it into a Jupyter notebook cell or running it from a Python script.
+This tutorial serves as an exploration of Vizro-AI beyond the initial [quickstart guide](../tutorials/quickstart.md). It explains the different language options and shows how to use Vizro-AI to enhance the formatting your visualizations.
 
+## Install Vizro-AI and get ready to run your code
+Before proceeding, you'll need to install Vizro-AI by following the steps in the [installation guide](../user-guides/install.md). To execute code either paste it into a Jupyter Notebook or run it from a Python script, or see the [guide on how to run Vizro-AI](../user-guides/run-vizro-ai.md) for more detail.
```

Comment:
Quick question as I also acknowledge that we have been quite inconsistent in the past - what should we include in a link? e.g.

- [user guide] on how to run Vizro-AI
- [guide on how to run Vizro-AI]
- guide on [how to run Vizro-AI]

Reply:
It doesn't matter too much. It's better to have more detail for people using screen readers (if they end up with a set of links that say "user guide" it doesn't help them navigate the docs much). I'd go with the full "go on how to run..." as per my edit here, but the third option is also fine.